lib/StatusIndicator/styles.scss

```diff
@@ -0,0 +1,39 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+
+$size-status-indicator: 1em !default;
+$cursor-status-indicator--hover: pointer !default;
+$color-status-indicator-label--text: rgb(111, 121, 129) !default;
+
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+
+.status-indicator__circle {
+  display: inline-block;
+  vertical-align: middle;
+  border-radius: 50%;
+  width: $size-status-indicator;
+  height: $size-status-indicator;
+  margin-right: .5rem;
+  margin-left: .25rem;
+}
+
+.status-indicator {
+  display: inline-block;
+  vertical-align: middle;
+  color: $color-status-indicator-label--text;
+}
+
+.status-indicator__label {
+  margin-right: .25rem;
+}
+
+.status-indicator__children .dateTime {
```

Comment:
Aye, I don't think we should add the styling of the .dateTime, as it's not part of the component - it's just what we added as a child in the stroybook as an example